auth.py

```diff
@@ -14,29 +17,42 @@ def authenticate(username, password, pwdb):
     else:
         ans = input(f"Would you like to add a new user with this name {username}? (y/n)")
         if ans == "y":
-            pwdb = add_user(pwdb, username, password, "pwdb.json")
+            pwdb = add_user(pwdb, username, password)
 
     return status
 
-def read_pwdb(path):
+def get_pwdb_path():
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), "pwdb.json")
```

Comment:
that is not going to work once our authentication system is installed. The installation directory may be read-only (in case you are working with system python, for example, or if the installation is done by someone else). so, it is better to use some path relative to the user directory